dataset/dictionary/items.json5

```
@@ -2126,54 +2126,67 @@
   {
     en: "Medaka",
     ja: "グッピー",
+    zhCN: "花鳉",
     tags: [ "item" ],
   },
   {
     en: "Glaze Medaka",
     ja: "琉璃グッピー",
+    zhCN: "琉璃花鳉",
     tags: [ "item" ],
   },
   {
     en: "Sweet-Flower Medaka",
     ja: "スイートグッピー",
+    zhCN: "甜甜花鳉",
     tags: [ "item" ],
   },
   {
     en: "Aizen Medaka",
     ja: "藍染グッピー",
+    zhCN: "蓝染花鳉",
     pronunciationJa: "あいぜんグッピー",
     tags: [ "item" ],
   },
   {
     en: "Crystalfish",
     ja: "水晶宴",
+    zhCN: "水晶宴",
     tags: [ "item" ],
   },
+  // Note: seems missing Dawncatcher, a type of Medaka that can be found as a pet for the sereniteapot
+  // Also, many other types of fishes are missing, you may need to complete them later.
+  // zhCN: "擒霞客"
```

Unfortunately, I don't play fishing so much, so I don't have so many kinds of fish. That's why there are many missing fishes.
You can add the words if you know the Chinese-English translations of fish. Japanese translation is not the required parameter.

```suggestion
  {
    en: "Dawncatcher",
    zhCN: "擒霞客",
    tags: [ "item" ],
  },
```

---

Okay, I will do it after this PR